frontend/src/containers/dialogs/programs/ActivateProgram.tsx

```diff
@@ -79,6 +79,10 @@ export const ActivateProgram = ({
         <DialogContent>
           <DialogDescription>
             {t('Are you sure you want to activate this Programme?')}
+            <br />
+            {t(
+              'Upon activation of this Programme a default Programme Cycles will be created.',
```

Comment:
`              'Upon activation of this Programme, a default Programme Cycle will be created.',
`
or
`              'Upon activation of this Programme, default Programme Cycles will be created.',
`